stable/xray/templates/xray-rabbitmq-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if index .Values "rabbitmq" "enabled" }}
```

Comment:
Why index? Why not just if .Values.rabbitmq.enabled ?

Reply:
Fixed